extra-dev/packages/coq-color/coq-color.8.7.dev/opam

```diff
@@ -71,7 +70,7 @@ tags: [
   "keyword:string"
 
   "keyword:alpha-equivalence"
-  "keyword:de Bruijn indices"
+  "keyword:de bruijn indices"
```

Comment:
Hum, this is a name... are you sure?

Reply:
I went by the majority rule. Feel free to commit a huge patch to capitalize all the keywords involving names.